src/TermInterface.jl

```diff
@@ -64,8 +74,11 @@ end
 
 Returns a term that is in the same closure of types as `typeof(x)`,
 with `head` as the head and `args` as the arguments, `type` as the symtype
-and `metadata` as the metadata. By default this will execute `head(args...)`
+and `metadata` as the metadata. By default this will execute `head(args...)`.
+`x` parameter can also be a `Type`.
 """
-similarterm(x, head, args, type; metadata=nothing) = head(args...)
+similarterm(x, head, args; type=nothing, metadata=nothing) = head(args...)
```

Comment:
`type` is a keyword argument here but not in the next line?

Reply:
fixed